functions.php

```diff
@@ -7,10 +7,10 @@
 /**
  * Enqueue an action to run one time, as soon as possible
  *
- * @param string $hook The hook to trigger.
- * @param array  $args Arguments to pass when the hook triggers.
- * @param string $group The group to assign this job to.
- * @return string The action ID.
+ * @param string $hook The hook to trigger
```

Comment:
The trailing periods should be left for this block to pass the sniffs.

Reply:
There's no consistency on phpDoc blocks: some blocks [have periods](https://github.com/woocommerce/action-scheduler/blob/be0a27922b6ca34f5f5e924cb7265b78062a2918/functions.php#L10), [some do not](https://github.com/woocommerce/action-scheduler/blob/be0a27922b6ca34f5f5e924cb7265b78062a2918/functions.php#L22). 

If a decision is made on which should be used, do not mind making that happen.

Reply:
> If a decision is made on which should be used, do not mind making that happen.

The current standard is ending period. But, the way PHPDocs formatting changes are addressed in any project I've worked on is to update a file when making other changes in that file.

Reply:
@rrennick I've added periods in cdfbd42.